stubs/psutil/psutil/__init__.pyi

```diff
@@ -92,14 +92,16 @@ if sys.platform == "win32":
         IOPRIO_LOW as IOPRIO_LOW,
         IOPRIO_NORMAL as IOPRIO_NORMAL,
         IOPRIO_VERYLOW as IOPRIO_VERYLOW,
+        win_service_get as win_service_get,
         win_service_iter as win_service_iter,
     )
 
 if sys.platform == "linux":
     PROCFS_PATH: str
-AF_LINK: Any
+AF_LINK: int
```

Comment:
This is an enum for me on 5.4.2:
```
In [74]: psutil.AF_LINK
Out[74]: <AddressFamily.AF_LINK: 17>
```

Reply:
This is one of the hardest packages I've ever typed 😒 
Types are so heavily dependent on platforms! It is `int` for me on macos.

I will check this once again. Thanks!

Reply:
I just checked their codebase and it seems to set this attribute in tons of different ways :( The enum on Linux is an IntEnum, so maybe we should just stick with `int` as the common type.

Reply:
Sounds like a good idea for now. It can always be improved.